### PR TITLE
Added Phonetics Tables

### PR DESCRIPTION
**Pull request by ollyturner**

### CODE REVIEW

File: LANGUAGE/Common Speech/Phonetics Table.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+Combination of Letters|Sound
```
Comment (mrcoolepicman) [inline LANGUAGE/Common Speech/Phonetics Table.md:1]: I recommend you make it plural, as the letters create multiple sounds.

**Review by mrcoolepicman — COMMENTED** [review]
Mostly good. Just make 'sound' on line 1 of file 2 plural.

File: LANGUAGE/Common Speech/Phonetics Table.md (hunk 1 above)
Comment (ollyturner) [reply LANGUAGE/Common Speech/Phonetics Table.md]: I have done that. Please review it again.

**Review by mrcoolepicman — COMMENTED** [review]
Perfect.

**Pull request merged**